lib/src/core/services/networking/api_service_depecrated.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:convert';
```

Comment:
If this file is deprecated is better to delete it before merging to master

Reply:
I would like to double check which implementation for service are we using, this would be an alternative to dio_service